test/OpenTelemetry.Tests/Implementation/Trace/Samplers/ProbabilityActivitySamplerTest.cs

```diff
@@ -0,0 +1,212 @@
+// <copyright file="ProbabilityActivitySamplerTest.cs" company="OpenTelemetry Authors">
```

Comment:
This is the same test code already used by `ProbabilitySampler` slightly modified to used the `Activity` equivalent types.